src/libs/GroupChatUtils.ts

```diff
@@ -6,8 +6,11 @@ import * as ReportUtils from './ReportUtils';
 /**
  * Returns the report name if the report is a group chat
  */
-function getGroupChatName(report: OnyxEntry<Report>): string | undefined {
-    const participants = report?.participantAccountIDs ?? [];
+function getGroupChatName(report: OnyxEntry<Report>, shouldApplyLimit = false): string | undefined {
+    let participants = report?.participantAccountIDs ?? [];
```

Comment:
Let's apply a sort order so it's consistent? (h/t @Pujan92)
```suggestion
    let participants = report?.participantAccountIDs.sort() ?? [];
```

Reply:
If it seem sorted by default then that is fine 👍

Reply:
I don't see that it is always sorted and in the same order, @dragnoir plz check again.

https://github.com/Expensify/App/assets/14358475/9029b253-11fb-4c96-83b8-464ba082bb88

> Let's apply a sort order so it's consistent? 

@deetergp Other option is to apply limit once the participants sorted with the display name here to see them in alphabetical order.

https://github.com/Expensify/App/blob/0aa0ffa8301d4d8bdb43f4a6985098b437428f47/src/libs/GroupChatUtils.ts#L10-L20

Reply:
We need to apply sort then both(web & mobile) the places on `participantAccountIDs`, but let @deetergp confirm because I am seeing the same(without initial sort) implementation for `ReportUtils.getReportName`